### ISSUE TITLE
Change Tab highlight 

### ISSUE THREAD

**Issue by denebasecas** [ogaclejapan/SmartTabLayout#13]
Hi Ogaclejapan,

I am trying to change the color of the tab when I Click on it. It shows blue as default. It is very UGLY

![sin titulo](https://cloud.githubusercontent.com/assets/6497625/7339831/e8ca9058-ec45-11e4-9fa1-e5aeff0bf150.png)

If I want to use pink or transparent as my default highlight, what could I do?, please.

Thanks in advance, and sorry for my lack of knowledge.

**Comment by ShkurtiA** [comment]
Hello,
I had the same problem because I am using this library but not with a `ViewPager`. I also needed to remove that click so I have customized the library. To fix that error inside `SmartTabLayout`  at method `populateTextTabStrip` I added at line 364: 
tabView.setBackgroundColor(tabView.getResources().getColor(android.R.color.transparent)); 
so that I can override the default click selector.

**Comment by ogaclejapan** [comment]
Hi, @denebasecas @ShkurtiA 

In the current version must define the layout of the custom tab.


res/drawable/tab_background_selector.xml

```xml

<?xml version="1.0" encoding="utf-8"?>
<selector xmlns:android="http://schemas.android.com/apk/res/android">
    <item android:state_pressed="true">
        <shape android:shape="rectangle">
            <solid android:color="@color/your_highlight_color"/>
        </shape>
    </item>
    <item android:drawable="@color/your_normal_color"/>
</selector>

```


res/layout/custom_tab.xml

```xml

<?xml version="1.0" encoding="utf-8"?>
<TextView xmlns:android="http://schemas.android.com/apk/res/android"
    android:id="@+id/custom_text"
    android:layout_width="wrap_content"
    android:layout_height="match_parent"
    android:textSize="14sp"
    android:textColor="@color/your_tab_text_color"
    android:gravity="center"
    android:paddingLeft="24dp"
    android:paddingRight="24dp"
    android:background="@drawable/tab_background_selector"
    >

</TextView>

```


res/layout/your_layout.xml

```xml

...

<com.ogaclejapan.smarttablayout.SmartTabLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:id="@id/viewpagertab"
    android:layout_width="match_parent"
    android:layout_height="@dimen/tab_height"
    ...
    app:stl_customTabTextLayoutId="@layout/custom_tab"
    app:stl_customTabTextViewId="@id/custom_text"
    />

...

```


Try implemented in the next version so that it can be easily specified in the attributes of SmartTabLayout :+1:

**Comment by ogaclejapan** [comment]
Hi, @denebasecas @ShkurtiA
Right now, released the 1.1.3 :+1: 

res/layout/your_layout.xml

```java

...

<com.ogaclejapan.smarttablayout.SmartTabLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:id="@id/viewpagertab"
    android:layout_width="match_parent"
    android:layout_height="@dimen/tab_height"
    ...
    app:stl_defaultTabBackground="@drawable/your_tab_background_selector"
    />

...

```
